### ISSUE TITLE
To use on own server

### ISSUE THREAD

**Issue by anacshora** [allancoding/emulatorjs-maker#6]
I'm new here, sorry for the question but, can emulatorjs-maker be used on my own server or is it just for use at https://allancoding.github.io/emulatorjs-maker ?

**Comment by allancoding** [comment]
I can be used but what part do you what on your server?

**Comment by anacshora** [comment]
Yes, I have a server with php and Nginx, and another server with php and Apache. Currently I'm testing the emulators on the first server.

**Comment by allancoding** [comment]
Ok because it will use php to store and upload files to the server.

**Comment by anacshora** [comment]
Ok, thanks in advance.

**Comment by navarrofnc** [comment]
> I'm testing emulatorjs-allancoding and emulatorjs-server (both have their own installation instructions) on my Ubuntu server and I liked the functionality. I've seen that it's easier to create the games with emulatorjs-maker, I haven't found the installation instructions, so I'd like to know if it's possible to implement the code together with emulatorjs-allancoding and emulatorjs-server on my own server, or if i can only use it on your website. I thank you in advance.

Hello, did you manage to implement the maker on your server?

**Comment by allancoding** [comment]
I am almost done.

**Comment by anacshora** [comment]
> I am almost done.

I'm looking forward to it, thank you very much.

**Comment by anacshora** [comment]
> I am almost done.

Hi, how are the preparations?
